### ISSUE TITLE
Mention resource limits / guarantees

### ISSUE THREAD

**Issue by minrk** [jupyterhub/binder#14]
We should mention resource limits and guarantees in the docs, pointing to [z2jh docs](https://zero-to-jupyterhub.readthedocs.io/en/latest/user-experience.html#set-user-memory-and-cpu-guarantees-limits) on the matter.

mybinder.org currently uses 1G guarantee and 4G limit:

```yaml
jupyterhub:
  singleuser:
    memory:
      guarantee: 1G
      limit: 4G
```

**Comment by choldgraf** [comment]
+1 ... we should also use this to make a pitch for people rolling their own binderhubs if they want to allow users to have more computational resources than this

**Comment by choldgraf** [comment]
I believe this was closed by #15
